appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java

```diff
@@ -71,7 +75,12 @@
   private boolean dialogVisible = false;
   boolean fromBackground = false;
   private Handler handleRetryConnection = new Handler();
-  private int connectionRetrys = 0;
+  private int connectionRetries = 0;
+  private boolean isWalletInstalled;
+  private AppcoinsAdvertisementRepository appcoinsAdvertisementRepository;
+  private AppcoinsAdvertisementConnection appcoinsAdvertisementConnection;
+  private static boolean showPopUpNotification;
```

Comment:
Remove not used and join the new static variables with the already existing static variables